rst2pdf/extensions/inkscape_r2p.py

```diff
@@ -58,10 +50,12 @@ def __init__(
         cache = self.source_filecache.setdefault(client, {})
         pdffname = cache.get(filename)
         if pdffname is None:
+            progname = shutil.which('inkscape')
```

Comment:
Does this work on a stock Windows with Inkscape installed? The [docs](https://docs.python.org/3/library/shutil.html#shutil.which) imply that it searches the path and I'm surprised that `$PROGRAMFILES\Inkscape` is on the path in Windows by default. Does the Windows Inkscape installer add this directory to the path?

i.e. I'd expect this change to mean that it no longer works "out of the box" on Windows.

Reply:
[This answer on SO](https://stackoverflow.com/questions/56399696/shutil-which-not-finding-programs-without-appending-file-extension) suggests the answer is yes. I don't have a Windows installation to test on though so I think we need to find someone that does else chance it.

Reply:
I fired up a Windows VM, installed Inkscape and Python and tested. The answer is that the `inkscape.exe` has to be on the path for `shutil.which()` to find it and by default, the Inkscape installer doesn't add its bin directory to the path.

![bf614322](https://user-images.githubusercontent.com/33135/83971030-f082f680-a8d0-11ea-9eee-9505e9178b28.png)
 
Incidentally, Inkscape v1's installer places the binary in `C:\Program Files\Inkscape\bin`, so the current default doesn't work going forwards anyway.

Reply:
Essentially, I think we just have to document that Inkscape's binary needs to be on your path.